templates/plugin/python/module.conf.erb

```diff
@@ -1,7 +1,7 @@
   Import "<%= @module %>"
 
   <Module "<%= @module %>">
-  <% @config.sort.each do |key,value| -%>
-    <%= key -%> <%= value -%>
-  <% end %>
+  <%- @config.sort.each do |key,value| -%>
+    <%= key %> <%= value %>
+  <%- end -%>
```

Comment:
Please replace this by `<% end %>`, otherwise there won't be new line between multiple modules.